src/platforms/posix/drivers/df_mpu9250_wrapper/df_mpu9250_wrapper.cpp

```diff
@@ -133,11 +136,22 @@ class DfMpu9250Wrapper : public MPU9250
 		float z_scale;
 	} _gyro_calibration;
 
+	struct mag_calibration_s {
+		float x_offset;
+		float x_scale;
+		float y_offset;
+		float y_scale;
+		float z_offset;
+		float z_scale;
+	} _mag_calibration;
+
 	int			    _accel_orb_class_instance;
 	int			    _gyro_orb_class_instance;
+	int			    _mag_orb_class_instance;
 
 	Integrator		    _accel_int;
 	Integrator		    _gyro_int;
+	Integrator		    _mag_int;
```

Comment:
You are correct.  This wouldn't make sense for mag values since they are not integrated.